programs/stake/src/stake_state.rs

```diff
@@ -209,7 +215,14 @@ fn calculate_stake_points(
         true, // this is safe because this flag shouldn't affect the first
               // element (i.e. `points`) of the returned tuple in any way
     )
-    .0
+    .points
+}
+
+#[derive(Debug, PartialEq)]
+struct StakePointsEarned {
```

Comment:
bikeshed: `EarnedStakePoints` or `RewardedStakePoints` or `CalculatedStakePoints`?  (it seems that `earned` and `rewarded` are already co-existing...)

I want to strive for normal english grammartical order for overall consystency across this mono-repo. speaking of which, i think `credits_observed` is a outlier and should be `observed_credits` actually....

Reply:
I agree, but I think we're kind of locked into `credits_observed` in the `Stake` struct. I played with different options for the credits field in these structs, but ultimately decided to keep the word order consistent with Stake: `new_credits_observed`